ecosystem/sep-0006.md

```diff
@@ -987,7 +987,8 @@ Name | Type | Description
 `stellar_transaction_id` | string | (optional) transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal.
 `external_transaction_id` | string | (optional) ID of transaction on external network that either started the deposit or completed the withdrawal.
 `message` | string | (optional) Human readable explanation of transaction status, if needed.
-`refunded` | boolean | (optional) Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
+`refunded` | boolean | (**deprecated**, optional) This field is deprecated in favor of the `refund` object. Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
+`refund` | object | (optional) An object describing any on or off-chain refund associated with this transaction. The schema for this object is defined in the [Refund Object Schema](#refund-object-schema) section below.
```

Comment:
I think we should name this field `refunds` because it is going to provide us with information about related refunds, and their aggregate affect. The name `refund` implies to me that it contains a single refund.